example.rb

```diff
@@ -8,10 +8,15 @@
 WEST_US = 'westus'
 GROUP_NAME = 'azure-sample-compute-group'
 
-StorageModels = Azure::ARM::Storage::Models
-NetworkModels = Azure::ARM::Network::Models
-ComputeModels = Azure::ARM::Compute::Models
-ResourceModels = Azure::ARM::Resources::Models
+Storage = Azure::Storage::Profiles::Latest::Mgmt
```

Comment:
should we use our defined profile for this, instead of Latest?

Reply:
I think Latest should be fine to maintain consistency for now. Once we make a decision on snapshots, we may need to update the samples.